CONTRIBUTING.md

```diff
@@ -0,0 +1,83 @@
+       </picture>
+   </a>&nbsp;&nbsp;&nbsp;
+   <a href="http://revanced.app/discord">
+       <img height="24px" src="https://user-images.githubusercontent.com/13122796/178032563-d4e084b7-244e-4358-af50-26bde6dd4996.png" />
+   </a>&nbsp;&nbsp;&nbsp;
+   <a href="https://reddit.com/r/revancedapp">
+       <img height="24px" src="https://user-images.githubusercontent.com/13122796/178032351-9d9d5619-8ef7-470a-9eec-2744ece54553.png" />
+   </a>&nbsp;&nbsp;&nbsp;
+   <a href="https://t.me/app_revanced">
+       <img height="24px" src="https://user-images.githubusercontent.com/13122796/178032213-faf25ab8-0bc3-4a94-a730-b524c96df124.png" />
+   </a>&nbsp;&nbsp;&nbsp;
+   <a href="https://x.com/revancedapp">
+      <picture>
+         <source media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/93124920/270180600-7c1b38bf-889b-4d68-bd5e-b9d86f91421a.png">
+         <img height="24px" src="https://user-images.githubusercontent.com/93124920/270108715-d80743fa-b330-4809-b1e6-79fbdc60d09c.png" />
+      </picture>
+   </a>&nbsp;&nbsp;&nbsp;
+   <a href="https://www.youtube.com/@ReVanced">
+       <img height="24px" src="https://user-images.githubusercontent.com/13122796/178032714-c51c7492-0666-44ac-99c2-f003a695ab50.png" />
+   </a>
+   <br>
+   <br>
+   Continuing the legacy of Vanced
+</p>
+
+# 👋 Contribution guidelines
+This document describes how to contribute to ReVanced Manager.
+
+## 📖 Resources to help you get started
+* The [documentation](/docs/README.md) provides steps to build ReVanced Manager from source
+* Our [backlog](https://github.com/orgs/ReVanced/projects/12) is where we keep track of what we're working on
+* [Issues](https://github.com/ReVanced/revanced-manager/issues) are where we keep track of bugs and feature requests
+
+## 🙏 Submitting a feature request
+Features can be requested by opening an issue using the
+[feature request issue template](https://github.com/ReVanced/revanced-manager/issues/new?assignees=&labels=feature-request&projects=&template=feature-issue.yml&title=feat%3A+%3Ctitle%3E).
+
+> [!NOTE]
+> We may reject your request at the discretion of ReVanced Manager's maintainers,
+> please provide good motivation for a request to be accepted.
+
+## 🐞 Submitting a bug report
+If you encounter a bug while using the ReVanced Manager app, open an issue using the
+[bug report issue template](https://github.com/ReVanced/revanced-manager/issues/new?assignees=&labels=bug&projects=&template=bug-issue.yml&title=bug%3A+%3Ctitle%3E).
+
+## 📝 How to contribute
+> [!NOTE]
+> It's worth noting that we recommend that you discuss your changes with 
+> the maintainers of ReVanced Manager before contributing. 
+> This will help you determine whether your change is acceptable.
+
+1. Fork the repository and create your branch from `dev`
+2. Make sure that the commit name are clear and concise as possible and commit your changes.
```

Comment:
Not sure what this is supposed to mean.

Reply:
The commit name has to be clear to the maintainer to avoid confusion during development.

We wouldn't want something like `refactor: update stuff` because it's ambiguous.

```suggestion
2. Commit your changes and avoid using ambiguous name
```